sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/UserRepositoryDBTest.java

```diff
@@ -60,46 +54,34 @@ public void before() {
     }
 
     @Test
-    public void findOrFailUserByEmailAddress_user_found_by_email_address() {
+    void findOrFailUserByEmailAddress_user_found_by_email_address() {
         /* execute */
         User user = userRepository.findOrFailUserByEmailAddress("db_test_testuser1@example.org");
 
         /* test */
-        assertEquals(user1, user);
+        assertThat(user).isEqualTo(user1);
     }
 
     @Test
-    public void findOrFailUserByEmailAddress_user_NOT_found_by_email_address() {
-        /* execute */
-        NotFoundException exception = assertThrows(NotFoundException.class,
-                () -> userRepository.findOrFailUserByEmailAddress("db_test_testuser_not_existing@example.org"));
-
-        /* test */
-        assertEquals("No user with email address 'db_test_testuser_not_existing@example.org' found!", exception.getMessage());
+    void findOrFailUserByEmailAddress_user_NOT_found_by_email_address() {
+        /* execute + test */
+        assertThatThrownBy(() -> userRepository.findOrFailUserByEmailAddress("db_test_testuser_not_existing@example.org")).isInstanceOf(NotFoundException.class)
+                .hasMessage("No user with email address 'db_test_testuser_not_existing@example.org' found!");
     }
 
     @Test
-    public void user_being_owner_of_project_can_NOT_be_deleted__instead_a_notaccetable_exception_is_thrown() {
-        /* test */
-        expected.expect(NotAcceptableException.class);
-        expected.expectMessage("still owner of a project");
-
+    void user_being_owner_of_project_can_NOT_be_deleted__instead_a_notaccetable_exception_is_thrown() {
         /* prepare */
         Project project = TestProjectCreationFactory.createProject("project_repo_test1", user1);
-
         entityManager.persist(project);
 
-        /* execute */
-        userRepository.deleteUserWithAssociations(user1.getName());
-
+        /* execute & test */
+        assertThatThrownBy(() -> userRepository.deleteUserWithAssociations(user1.getName())).isInstanceOf(NotAcceptableException.class)
+                .hasMessage("User db_test_testuser1 is 1 times still owner of a project! Move ownership before delete!");
```

Comment:
Just a remark:
`.hasMessage(...)` versus `.hasMessageContaining(...)`: 

It is okay to use has message ... but most times I would prefer `.hasMessageContaining(...)` and to check only a relevant part.

Why? Because of maintenance. If you change just a bit of the sentence - e.g. "Move owner ship before delete!" would be changed to "Move all ownerships before you deltee the user!" the test will fail. But the test shall only test that there is a not acceptable exception thrown which complains that still owner of project.

So an alternative would have been:
```suggestion
                .hasMessageContaining("db_test_testuser1 is 1 times still owner");
```
It is also possible to combine with relevant parts like:
```suggestion
                .hasMessageContaining("db_test_testuser1").hasMessageContaining("1 times still owner");
```

If you would write tests this way, they would become more easy to maintain and concentrate not on the exact test but on the text essence.